packages/appcd/src/main.js

```diff
@@ -7,6 +7,16 @@ import cli from './cli';
 
 cli.exec()
 	.catch(err => {
-		console.error(err.message);
-		process.exit(err.exitCode || 1);
+		const exitCode = err.exitCode || 1;
+
+		if (err.json) {
+			console.log(JSON.stringify({
+				code: exitCode,
+				result: err.toString()
+			}, null, 2));
+		} else {
+			console.error(err.message);
```

Comment:
Kinda not related. 

I ran into some problems with the standard of logging only `err.message` and tracking the error down, could we maybe use snooplogg to log the full error so when this hits users machines we're not having them editing the source to give us stacks